### PR TITLE
data:NonsoBarn

### PR DESCRIPTION
**Pull request by NonsoBarn**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7009"></a>

### CODE REVIEW

File: data/NonsoBarn.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Nonso Barn",
+  "type": "personal",
+  "bio": "Web Developer. Web3 enthusiast, Games & Tech aficionado"
```
Comment (iamthecloverly) [inline data/NonsoBarn.json:4]: "bio": "Web Developer. Web3 enthusiast, Games & Tech aficionado"

Comment (khushalarora) [inline data/NonsoBarn.json:4]: Please add a **'Comma'** at the end of this line :+1: 
```Suggestion
  "bio": "Web Developer. Web3 enthusiast, Games & Tech aficionado",
```

File: data/NonsoBarn.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/NonsoBarn",
+      "icon": "FaTwitter"
+    }
```
Comment (khushalarora) [inline data/NonsoBarn.json:10]: Also, add a **'Comma'** here :+1: 
```Suggestion
    },
```

File: data/NonsoBarn.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/NonsoBarn",
+      "icon": "FaGitHub"
+    }
```
Comment (khushalarora) [inline data/NonsoBarn.json:15]: Also, add a **'Comma'** here :+1: 
```Suggestion
    },
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @NonsoBarn, thank you for contributing :tada:

File: data/NonsoBarn.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+{
+      "name": "Let's Connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/nonsobarn/",
+      "icon": "FaGitHub"
```
Comment (RohanSasne) [inline data/NonsoBarn.json:19]: Change the icon from FAGitHub to FaLinkedin:

```suggestion
      "icon": "FaLinkedin"
```

File: data/NonsoBarn.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/NonsoBarn",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/NonsoBarn",
+      "icon": "FaGitHub"
```
Comment (RohanSasne) [inline data/NonsoBarn.json:14]: The h should be small in Github:
```suggestion
      "icon": "FaGithub"
```

**Review by RohanSasne — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good! 🎉

**Pull request merged**